tests/StubGeneratorTest.php

```diff
@@ -218,6 +260,9 @@ public function test_it_can_generate_a_complete_stub(): void
 
 Phar::mapPhar('test.phar');
 Phar::interceptFileFuncs();
+
+require 'phar://test.phar/.box/check_requirements.php';
+
 require 'phar://test.phar/index.php';
```

Comment:
there is a test missing with just the index & intercept to ensure there is not a double blank lines